pkg/yang/modules.go

```diff
@@ -200,6 +204,9 @@ func (ms *Modules) FindModule(n Node) *Module {
 // FindModuleByNamespace either returns the Module specified by the namespace
 // or returns an error.
 func (ms *Modules) FindModuleByNamespace(ns string) (*Module, error) {
+	ms.mu.Lock()
+	defer ms.mu.Unlock()
```

Comment:
Could you add a comment here that this is used to protect `byNS` from a race condition?